spf.go

```diff
@@ -109,7 +110,10 @@ var (
 // Default value for the maximum number of DNS lookups while resolving SPF.
 // RFC is quite clear 10 must be the maximum allowed.
 // https://tools.ietf.org/html/rfc7208#section-4.6.4
-const defaultMaxLookups = 10
+const (
+	defaultMaxLookups     = 10
+	defaultMaxVoidLookups = 2
```

Comment:
Would you mind moving the pre-existing comment inside and above the `defaultMaxLookups` constant, since it applies to it only, and adding a new one for `defaultMaxVoidLookups`?

The comment could be something simple like you already have below, like:

```
The default value for the maximum number of void DNS lookups while resolving SPF.
A void DNS lookup is one that returns either no results, or a "name error".
RFC recommends a maximum of 2.
https://datatracker.ietf.org/doc/html/rfc7208#section-4.6.4
```

What do you think?